src/imitation/rewards/reward_nets.py

```diff
@@ -213,11 +213,10 @@ def dtype(self) -> th.dtype:
 
 
 class RewardNetWrapper(RewardNet):
-    """An abstract RewardNet wrapping a base network.
+    """Abstract class representing a wrapper that modifies a `RewardNet`'s functionality.
 
-    Note: The wrapper will default to forwarding calls to `device`, `forward`,
-        `preproces`, `predict`, and `predict_processed` to the base reward net unless
-        explicitly overridden in a subclases.
+    In general `RewardNetWrapper`s should either override `ForwardWrapper`
```

Comment:
is 'override' the right word here? should it instead be 'subclass'? (real question)

Reply:
It should be "subclass". You create a subclass of a superclass and then _override_ its methods.